src/app/spotlightPage/page.tsx

```diff
@@ -0,0 +1,95 @@
+'use client';
+
+import Link from 'next/link';
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+
+import { fetchSpotlightTours } from '../../supabase/tours/queries';
+import NavBar from '../../components/userComponents/navBar/navBar';
+import { TourRow} from '../../types/types';
+
+/**
+ * @returns spotlights from tours table
```

Comment:
mention how this is the home page and that it dynamically loads the following parts of hte spotlights flow.  Add desc of the general spotlight flow here